docs/SOURCING-COMMITMENTS.md

```diff
@@ -0,0 +1,91 @@
+# Sourcing commitments for joins
+
+This guide is for Joinmarket 0.2+ ; it is not relevant for earlier versions.
```

Comment:
I think this line is not needed here, anyone reading this will be already running 0.2+. Wiki has different, as that didn't come together with releases.